src/adapters/supabase/helpers/comment.ts

```diff
@@ -25,10 +28,11 @@ export class Comment extends SuperSupabase {
       return;
     } else {
       //Create the embedding for this comment
-      const embedding = await this.context.adapters.openai.embedding.createEmbedding(commentBody);
-      const { error } = await this.supabase
-        .from("issue_comments")
-        .insert([{ id: commentId, commentbody: commentBody, issuebody: issueBody, embedding: embedding }]);
+      const embedding = await this.context.adapters.openai.embedding.createEmbedding(plaintext);
+      if (isPrivate) {
+        plaintext = "CENSORED";
```

Comment:
```suggestion
        plaintext = null;
```
Should be initialized as null though.